### PR TITLE
GTEST/MEM_BUFFER, GTEST/UCP: Fix valgrind issue

### LINKED ISSUE #4134: failure in host/test_ucp_mem_type_alloc_before_init.xfer/0 with UCX_RNDV_SCHEME=put_zcopy under valgrind
```
$ UCX_RNDV_SCHEME=put_zcopy GTEST_FILTER=host/test_ucp_mem_type_alloc_before_init.xfer/0 make test_valgrind
env LD_LIBRARY_PATH="/usr/lib64/mlnx_ofed/valgrind::" \
stdbuf -e0 -o0 valgrind --tool=memcheck --leak-check=full --track-origins=yes --fair-sched=try --num-callers=25 --error-exitcode=1 --child-silent-after-fork=yes --suppressions=../../contrib/valgrind.supp  /labhome/dmitrygla/work_auto/ucx
/test/gtest/gtest --gtest_filter=host/test_ucp_mem_type_alloc_before_init.xfer/0
==29583== Memcheck, a memory error detector
==29583== Copyright (C) 2002-2015, and GNU GPL'd, by Julian Seward et al.
==29583== Using Valgrind-3.12.0 and LibVEX; rerun with -h for copyright info
==29583== Command: /labhome/dmitrygla/work_auto/ucx/test/gtest/gtest --gtest_filter=host/test_ucp_mem_type_alloc_before_init.xfer/0
==29583==
[     INFO ] checking dcx: no
[     INFO ] checking ud: no
[     INFO ] checking udx: no
[     INFO ] checking rc: no
[     INFO ] checking rcx: no
[     INFO ] checking shm_ib: yes
[     INFO ] checking ugni: no
[     INFO ] checking self: yes
[     INFO ] checking tcp: yes
[     INFO ] checking all: yes
[     INFO ] checking all_rcdc: no
[     INFO ] checking self_rcx: yes
[     INFO ] checking rc_x: no
[     INFO ] checking ud_mlx5: no
[     INFO ] checking shm: yes
[     INFO ] checking rc_ud: no
[     INFO ] checking dc_ud: no
[     INFO ] checking no_ud_ud_x: yes
[     INFO ] Using random seed of 23748
[     INFO ]  Setting for valgrind: UCX_IB_RX_QUEUE_LEN=512
[     INFO ]  Setting for valgrind: UCX_IB_RX_BUFS_GROW=512
[     INFO ]  Setting for valgrind: UCX_MM_RX_BUFS_GROW=128
[     INFO ]  Setting for valgrind: UCX_IB_TX_QUEUE_LEN=128
[     INFO ]  Setting for valgrind: UCX_IB_TX_BUFS_GROW=64
[     INFO ]  Setting for valgrind: UCX_RC_TX_CQ_LEN=256
[     INFO ]  Setting for valgrind: UCX_CM_TIMEOUT=600ms
[     INFO ]  Setting for valgrind: UCX_TCP_TX_BUFS_GROW=512
[     INFO ]  Setting for valgrind: UCX_TCP_RX_BUFS_GROW=512
[     INFO ]  Setting for valgrind: UCX_TCP_RX_SEG_SIZE=16k
Note: Google Test filter = host/test_ucp_mem_type_alloc_before_init.xfer/0
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from host/test_ucp_mem_type_alloc_before_init
[ RUN      ] host/test_ucp_mem_type_alloc_before_init.xfer/0
==29583== WARNING: valgrind ignores shmget(shmflg) SHM_HUGETLB
==29583== Syscall param process_vm_writev(lvec[...]) points to uninitialised byte(s)
==29583==    at 0x77F2F8A: process_vm_writev (in /usr/lib64/libc-2.17.so)
==29583==    by 0x8776B43: uct_cma_ep_common_zcopy (cma_ep.c:110)
==29583==    by 0x8776B43: uct_cma_ep_put_zcopy (cma_ep.c:129)
==29583==    by 0x5873BCB: uct_ep_put_zcopy (uct.h:2332)
==29583==    by 0x5873BCB: ucp_rndv_progress_rma_put_zcopy_inner (rndv.c:734)
==29583==    by 0x5873BCB: ucp_rndv_progress_rma_put_zcopy (rndv.c:693)
==29583==    by 0x5878410: ucp_request_try_send (ucp_request.inl:171)
==29583==    by 0x5878410: ucp_request_send (ucp_request.inl:206)
==29583==    by 0x5878410: ucp_rndv_rtr_handler_inner (rndv.c:1002)
==29583==    by 0x5878410: ucp_rndv_rtr_handler (rndv.c:917)
==29583==    by 0x53C1E05: uct_iface_invoke_am (uct_iface.h:628)
==29583==    by 0x53C2532: uct_mm_iface_invoke_am (mm_iface.h:151)
==29583==    by 0x53C2532: uct_mm_iface_process_recv (mm_iface.c:208)
==29583==    by 0x53C29DE: uct_mm_iface_poll_fifo (mm_iface.c:254)
==29583==    by 0x53C2B52: uct_mm_iface_progress (mm_iface.c:278)
==29583==    by 0x583417C: ucs_callbackq_dispatch (callbackq.h:211)
==29583==    by 0x583A8A9: uct_worker_progress (uct.h:2203)
==29583==    by 0x583A8A9: ucp_worker_progress (ucp_worker.c:1892)
==29583==    by 0x792CD0: ucp_test_base::entity::progress(int) (ucp_test.cc:655)
==29583==    by 0x78FB29: ucp_test::progress(int) const (ucp_test.cc:165)
==29583==    by 0x78FF44: ucp_test::wait(void*, int) (ucp_test.cc:222)
==29583==    by 0x6C4163: test_ucp_mem_type_alloc_before_init_xfer_Test::test_body() (test_ucp_mem_type.cc:119)
==29583==    by 0x5367E5: ucs::test_base::run() (test.cc:276)
==29583==    by 0x5369A8: ucs::test_base::TestBodyProxy() (test.cc:302)
==29583==    by 0x6625B5: ucp_test::TestBody() (ucp_test.h:143)
==29583==    by 0x518E93: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514071: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3598)
==29583==    by 0x4FB3F0: testing::Test::Run() (gtest-all.cc:3635)
==29583==    by 0x4FBBCB: testing::TestInfo::Run() (gtest-all.cc:3812)
==29583==    by 0x4FC25B: testing::TestCase::Run() (gtest-all.cc:3930)
==29583==    by 0x502A4B: testing::internal::UnitTestImpl::RunAllTests() (gtest-all.cc:5802)
==29583==    by 0x51A271: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514ED3: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3598)
==29583==  Address 0xaa95fd0 is 0 bytes inside a block of size 10,000 alloc'd
==29583==    at 0x4C29BE3: malloc (vg_replace_malloc.c:299)
==29583==    by 0x5274C9: mem_buffer::allocate(unsigned long, ucs_memory_type) (mem_buffer.cc:48)
==29583==    by 0x52832A: mem_buffer::mem_buffer(unsigned long, ucs_memory_type) (mem_buffer.cc:232)
==29583==    by 0x6C532D: test_ucp_mem_type_alloc_before_init::init() (test_ucp_mem_type.cc:88)
==29583==    by 0x53626E: ucs::test_base::SetUpProxy() (test.cc:228)
==29583==    by 0x662575: ucp_test::SetUp() (ucp_test.h:143)
==29583==    by 0x518E93: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514071: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3598)
==29583==    by 0x4FB39D: testing::Test::Run() (gtest-all.cc:3630)
==29583==    by 0x4FBBCB: testing::TestInfo::Run() (gtest-all.cc:3812)
==29583==    by 0x4FC25B: testing::TestCase::Run() (gtest-all.cc:3930)
==29583==    by 0x502A4B: testing::internal::UnitTestImpl::RunAllTests() (gtest-all.cc:5802)
==29583==    by 0x51A271: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514ED3: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3598)
==29583==    by 0x501687: testing::UnitTest::Run() (gtest-all.cc:5416)
==29583==    by 0x52364A: RUN_ALL_TESTS() (gtest.h:20059)
==29583==    by 0x523534: main (main.cc:102)
==29583==  Uninitialised value was created by a heap allocation
==29583==    at 0x4C29BE3: malloc (vg_replace_malloc.c:299)
==29583==    by 0x5274C9: mem_buffer::allocate(unsigned long, ucs_memory_type) (mem_buffer.cc:48)
==29583==    by 0x52832A: mem_buffer::mem_buffer(unsigned long, ucs_memory_type) (mem_buffer.cc:232)
==29583==    by 0x6C532D: test_ucp_mem_type_alloc_before_init::init() (test_ucp_mem_type.cc:88)
==29583==    by 0x53626E: ucs::test_base::SetUpProxy() (test.cc:228)
==29583==    by 0x662575: ucp_test::SetUp() (ucp_test.h:143)
==29583==    by 0x518E93: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514071: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (gtest-all.cc:3598)
==29583==    by 0x4FB39D: testing::Test::Run() (gtest-all.cc:3630)
==29583==    by 0x4FBBCB: testing::TestInfo::Run() (gtest-all.cc:3812)
==29583==    by 0x4FC25B: testing::TestCase::Run() (gtest-all.cc:3930)
==29583==    by 0x502A4B: testing::internal::UnitTestImpl::RunAllTests() (gtest-all.cc:5802)
==29583==    by 0x51A271: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3562)
==29583==    by 0x514ED3: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (gtest-all.cc:3598)
==29583==    by 0x501687: testing::UnitTest::Run() (gtest-all.cc:5416)
==29583==    by 0x52364A: RUN_ALL_TESTS() (gtest.h:20059)
==29583==    by 0x523534: main (main.cc:102)
==29583==
common/test.cc:261: Failure
Failed
Got 3 valgrind errors during the test
[  FAILED  ] host/test_ucp_mem_type_alloc_before_init.xfer/0, where GetParam() = all (1204 ms)
[----------] 1 test from host/test_ucp_mem_type_alloc_before_init (1222 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (1302 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] host/test_ucp_mem_type_alloc_before_init.xfer/0, where GetParam() = all

 1 FAILED TEST
==29583==
==29583== HEAP SUMMARY:
==29583==     in use at exit: 9,447 bytes in 39 blocks
==29583==   total heap usage: 166,888 allocs, 166,316 frees, 19,352,489 bytes allocated
==29583==
==29583== LEAK SUMMARY:
==29583==    definitely lost: 0 bytes in 0 blocks
==29583==    indirectly lost: 0 bytes in 0 blocks
==29583==      possibly lost: 0 bytes in 0 blocks
==29583==    still reachable: 9,447 bytes in 39 blocks
==29583==         suppressed: 0 bytes in 0 blocks
==29583== Reachable blocks (those to which a pointer was found) are not shown.
==29583== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==29583==
==29583== For counts of detected and suppressed errors, rerun with: -v
==29583== ERROR SUMMARY: 3 errors from 1 contexts (suppressed: 0 from 0)
make: *** [test_valgrind] Error 1
```

Reproduced on hpc-test-node-upstream02

### PR DESCRIPTION
**Pull request by dmitrygx**
## What

Fixes valgrind issue

## Why ?

Fixes #4134

## How ?

1. Add `mem_buffer::set()` that does `memset`/`cudaMemset` depends on a memory type
2. Use it in UCP mem type test to initialize a send buffer

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12275/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/b4b24068a4e170b4ce3643038ed2673bd0c005f5/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/a7010f461f2adcb143e8a166951325e9131ba283/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/0bf9515e4b5ab03bd37c3a0f57c9adbaa9905c68/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/76564a840ce133f3d80d4358d9835d906169be67/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/8e23e05b7d80b7b042a39faf0a8ba14bd061f0d1/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/b4c79b53ae3b3b5834b9634ec8267e6138d4439f/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/ed6d4483a5e0166b7ae1970c4146a8b7529f2bff/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/984ce1a3ea03204835d261528b75ecde9e507c1c/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/f2f00b0d2354490ba857776e07319aafc9407b75/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/168319f90ce9f728800ea28b596ef77eb27a8852/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/32c2532cc0c5050eea0aa034917551abab7a1874/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/e1a497ad4385bd7492e3707b902477b6e8400087/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/ca8d3ff6e30e6f0d07b4b19685393161d6c4e8f7/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/354195bdfa4128eb4c346663b09fdd29e8c1ce46/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/3714eb95b7ec16c80d813b442c556f3d959d4a8b/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/0dfa39d8a2fccf6fd2a1a7fe3f3d84cdbd156d1a/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/0eb720a8233f25dbd6fcf3625b1778335353c615/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/d945c1b26f3b29387ff69c1c2888dfcba084e4e1/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/3d0f515d3be6e20af48911f675bca8578c84b4f8/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/9574f9c66fa6e1ddd21e13740ef5d78d39fe59ff/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/9ea4606f4907d0a3fa0e2796d63a70c03d161cd4/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/d710b55ceb62fb7ec2d91a27962eef45a5d9dca1/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/172990971c1b97ad4934806e0be63f3ae6c721b0/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/585b83020d07f35fd85d748d5ba3b7753d19b632/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/5626eb738cd7d129bdb2dec00a47c53f25a72cb4/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/6127d5d69632bd0edda06af2b92217cac8a5e8a9/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/23d313398d9ac1ed6af4327a38e8c8ce14bca2e5/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/53e2dd6e5bcbdd0760c5b0d85dc9f5eaf960155c/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c0a7d440b9b599d5f15310cd40f273e2/raw/f99599f9be2c018c344314e153714ab97174f8dd/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: test/gtest/ucp/test_ucp_mem_type.cc [hunk 1]
```diff
@@ -87,6 +87,9 @@ class test_ucp_mem_type_alloc_before_init : public test_ucp_mem_type {
     virtual void init() {
         m_send_buffer.reset(new mem_buffer(m_size, mem_type()));
         m_recv_buffer.reset(new mem_buffer(m_size, mem_type()));
+
+        m_send_buffer->set(m_send_buffer->ptr(), 0, m_size, mem_type());
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_mem_type.cc]: maybe use pattern_fill + pattern_check ?

Comment (dmitrygx) [reply test/gtest/ucp/test_ucp_mem_type.cc]: yep, this is good idea

Comment (dmitrygx) [reply test/gtest/ucp/test_ucp_mem_type.cc]: > maybe use pattern_fill + pattern_check ?

@yosefe done, CUDA support is ok there

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12297/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 16-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/6cce8049b85156b47552e5f077c53cd1077a4cd4/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/c818631be1d7f9c4c21d598b04f5c13060b03e3d/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/b65521cb0b50ea476930db77e5d301cdbb441888/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/7ba1f8df90ee97e102a0c6a3fb1b9ddf15ccba26/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/519942432b1d79d88131ccd4b9e749ab8b6f3c20/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/54f06d861c3ad624a7d8a9979210990cad85705f/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/d03fe1907748e7d0cebd4cf8e70bec56cc73bbc2/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/463aa32ae86fc8723e7113c57bd2a4920dc535af/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/a288a02ab70903b9d1e264abbbf2f39d27314606/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/765326273a3d04d71832d7bfdd31628a2d6c9de9/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/637b7880ede002876606f765bd98cd71cb4f82ea/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/27b6e1af3c1cf8765a90b1731b42f72182b76f2d/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/960a1ded5e9963b3d6123f5f7b9df0338642ae3f/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/9702f3dca780aa84bbf9d8f0caaab0c48a6585f4/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/9433ecd115feffccd5b02ff454f3570fbf52c6a2/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/640c24bd2365aabd565a9529e8636ae5e1d007c3/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/479ba53be8648070a1fd17f5127a4fb4ef8fab4a/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/288c181757b23426016b4e34dfee7a8cb0fb6218/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/ba474677ee9a136bba734122c10dbc6eb7008c32/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/1ffa18c968fc91d65d66541f83ca5f188632e0c2/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/bdd3a3a299e428e1fa2a45f1e0f1918bd9d38030/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/53be67795e47dbb2eba7c6291ac974d45ddc2a4a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/0447c004d039703f441bd89ae48d91372fdb2810/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/7cb0b9ed91da90c7b7afc867c0aa37e8439c48e4/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/e4aec747659dae1bd763c18ee25a46398d0acc85/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/b6f22a229f1ecb835be179fcf2412ac27cfb6947/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/0005d237b9689116d32df921a8e098b0ae72b793/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/f39291182863ffcd099908e5cadcf9916bf4ee05/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c5296143be70290e332fabe64c040075/raw/a8396845fc522160d8adf7ad04242d2dcab7b607/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by dmitrygx** [comment]
@yosefe could you review pls?

**Pull request merged**